packages/lesswrong/themes/eaTheme.ts

```diff
@@ -301,11 +301,6 @@ const theme = createLWTheme({
         color: grey[800]
       }
     },
-    CommentPermalink: {
```

Comment:
My guess is this shouldn't be in this PR? But sure, it's your theme.

Reply:
It's because it have errors (because it didn't have a style to override) which is kinda like logging.